source/_docs/guides/jenkins.md

```diff
@@ -0,0 +1,223 @@
+
+```nohighlight
+git remote add origin git@github.com:YOUR-ORG/YOUR-PROJECT.git
+git push -u origin master
+```
+
+At this point, you should be able to create a local branch, commit a change, and push to GitHub. On GitHub, you should then be able to open a pull request successfully.
+
+## Step 3: Create a Pantheon Site
+
+Now we will spin up a Drupal 8 site on Pantheon with Terminus, then overwrite the default install with the code from our GitHub repo.
+
+1. From your local terminus, use terminus to create a site on Pantheon:
+```nohighlight
+terminus site:create $SITE "My Site" "Drupal 8" --org="My Team"
+terminus connection:set $SITE.dev git
+```
+
+2. Add the Pantheon remote repository address and push the code to it.
+```nohighlight
+PANTHEON_REPO=$(terminus connection:info $SITE.dev --field=git_url)
+git remote add pantheon $PANTHEON_REPO
+git push --force pantheon master
+
+```
+
+3. Then complete the installation on Pantheon.
+
+```nohighlight
+terminus build-env:site-install --site-mail="<your email>" --site-name="My Drupal Site" --account-mail="<your email>" --account-name="admin" $SITE.dev
+```
+
+5. Verify the site is installed and working.
+```nohighlight
+terminus dashboard:view $SITE
+```
+
+Now the master branch of GitHub, your local, and Pantheon are in sync.
+
+## Step 4: Create Jenkins Project #1
+This Project tracks changes to all branches except the master branch. We will clone this to create a second project which tracks only the master branch changes and pushes the code to Pantheon. 
+
+1. Log into the Jenkins dashboard as an admin user. Click "New Item."
+
+![Jenkins dashboard](/source/docs/assets/images/integrations/jenkins/jenkins_dash.png)
+
+2. Give the project a name. Do not use spaces. Select "Freestyle Project" and "OK" to save.
+
+![New Jenkins project](/source/docs/assets/images/integrations/jenkins/new_job.png)
+
+3. In the General section, check the box to select "GitHub project" and enter the repo url (e.g. https://github.com/YOUR-ORG/YOUR-PROJECT).
+
+4. In the Source Code Management section, add:
+- Repository URL: The path to the .git file of your repository, e.g. https://github.com/YOUR-ORG/YOUR-PROJECT.git 
+- Name: origin
+- Refspec: +refs/heads/*:refs/remotes/origin/*
+- Branches specifier: origin/*
+- Additional Behaviours: Prune stale remote-tracking branches, and Clean after checkout
+
+![SCM view](/source/docs/assets/images/integrations/jenkins/scm_settings.png)
+
+5. We want code changes to trigger our build, as opposed to setting up a periodic build, for example. Check the box labelled, "GitHub hook trigger for GITScm polling."
+
+6. For "Build Environment," check the box labelled, "Inject environment variables to the build process." Verify the file path exists on the Jenkins server, or create a file and add the path accordingly. Add these variables to the "Properties Content" field, one per line, with no quotations marks.
```

Comment:
I'd check and see what Ol' Jenky is doing by default. IIRC the old "watch this repo and build on changes" only actually builds on commits, not PRs, and might even need branch names defined in advance. We should test this. 

We might actually want to favor a webhook model of triggered builds, in which case the webhook certainly contains the data.

More than anything, let's talk to real users doing this already and see what they think.